src/style.css

```diff
@@ -0,0 +1,120 @@
+body {
```

Comment:
There is nothing esle to say.
Optional 
You should more styling for styling for your Css